### ISSUE TITLE
When changing the earth world the rocket gui isnt able to be opened anymore

### ISSUE THREAD

**Issue by marvjosh1** [Slimefun-Addon-Community/Galactifun#107]
## Description (Required)
Whenever I tried changing the earth world and trying to open a rocket, it doesnt open its GUI

## Steps to reproduce the Issue (Required)
https://youtu.be/5dT3EBWk2DA
https://i.imgur.com/g2eEYM8.png

## Expected behavior (Required)
The rocket GUI to open properly

## Server Log / Error Report
n/a

## Environment (Required)
https://i.imgur.com/x508Vfl.png
https://i.imgur.com/5qLHwRH.png

**Comment by marvjosh1** [comment]
Forgot to mention that this is a tier 3 rocket I just placed down 5 seconds before making the video

**Comment by Seggan** [comment]
Are you *on* the changed earth world?

**Comment by Seggan** [comment]
Did you restart the server after changing the setting?

**Comment by Seggan** [comment]
Also, any errors?

**Comment by marvjosh1** [comment]
Yes I restarted the server afterwards, and no, I did not get any errors

**Comment by Seggan** [comment]
Please don't comment on random pull requests. I cannot tell what is causing this. No one else has reported this.
